website/project/model.py

```diff
@@ -736,9 +115,10 @@ def validate_title(value):
 
 
 def validate_user(value):
```

Comment:
I'd probably move most of these into class methods on the models that they relate to. Django prefers fat models over helper methods.